aten/src/ATen/core/CachingHostAllocator.h

```diff
@@ -274,11 +369,127 @@ struct CachingHostAllocatorImpl {
     TORCH_CHECK_NOT_IMPLEMENTED(false, "Not implemented for copy_data");
   }
 
+  HostStats getStats() {
+    HostStats stats;
+
+    // Flush any available blocks into the free_list. Not strictly necessary,
+    // but it will yield more accurate stats.
+    // @REVIEW: do we want this here or not?
+    process_events();
```

Comment:
Better not, let's try to keep getStats lighter

Reply:
Sounds good, removed.